ehitchhiking-ui/src/app/pages/main-screen/trip-registration/trip-registration.component.ts

```diff
@@ -22,25 +24,35 @@ export class TripRegistrationComponent implements OnInit {
 	coords;
 	nameFormGroup: FormGroup;
 
-	constructor(private userService: UserService) {}
+	constructor(private userService: UserService, private yandexMapService: YandexMapService ) {}
 
 	ngOnInit() {
 		this.nameFormGroup = new FormGroup({
-			from: new FormControl('Барановичи', [Validators.required]),
-			to: new FormControl('Пинск', [Validators.required]),
+			from: new FormControl('', [Validators.required]),
+			to: new FormControl('', [Validators.required]),
 			departureDate: new FormControl('', [Validators.required]),
-			placesSelect: new FormControl('1', [Validators.required]),
+			placesSelect: new FormControl('', [Validators.required]),
 			departureTime: new FormControl('', [Validators.required]),
-			car: new FormControl(''),
+			car: new FormControl('', [Validators.required]),
```

Comment:
Are you sure that [Validators.required] is required for all cases? What about passengers form?

Reply:
😀